pages/common/az-disk.md

```diff
@@ -0,0 +1,25 @@
+# az disk
+
+> Manage Azure Managed Disks.
+> Part of `azure-cli` (also known as `az`).
+> More information: <https://learn.microsoft.com/cli/azure/disk>.
+
+- Create a managed disk:
+
+`az disk create --resource-group {{resource_group}} --name {{disk_name}} --size-gb {{size_in_gb}}`
+
+- List managed disks in a resource group:
+
+`az disk list --resource-group {{resource_group}}`
+
+- Delete a managed disk:
+
+`az disk delete --resource-group {{resource_group}} --name {{disk_name}}`
+
+- Grant access to a managed disk (for export):
+
+`az disk grant-access --resource-group {{resource_group}} --name {{disk_name}} --access-level Read --duration-in-seconds {{seconds}}`
```

Comment:
Does `--access-level` also accept other values? (if it does, maybe it should be `{{Read}}`?)

Reply:
```suggestion
`az disk grant-access --resource-group {{resource_group}} --name {{disk_name}} --access-level {{Read|Write}} --duration-in-seconds {{seconds}}`
```
It does -> https://learn.microsoft.com/pl-pl/cli/azure/disk?view=azure-cli-latest#az-disk-grant-access